content/api/ab-testing.apib

```diff
@@ -417,6 +417,41 @@ Returns the `status`.
             ]
         }
 
+### Delete an A/B Test [DELETE /ab-test/{id}]
+
++ Parameters
+    + id (required, string, `password-reset`)
+
++ Request
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 204 (application/json)
+
+
++ Response 404 (application/json)
```

Comment:
hmm - the actual behavior doesn't match this.  I just get:

```
{
  "errors": [
    {
      "message": "Resource could not be found"
    }
  ]
}
```

@awinder - I seem to remember you mentioning there's some nginx magic that sets the response body to the above for all 404s?